src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,29 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy.dao");
+
     public static void main(String[] args) {
+        BookDao bookDao = (BookDao) injector.getInstance(BookDao.class);
+        Book book = new Book();
+        book.setTitle("Effective Java");
+        book.setPrice(BigDecimal.valueOf(45.99));
+
+        bookDao.create(book);
+        System.out.println("Created: " + book);
+
+        bookDao.findAll().forEach(System.out::println);
+
+        book.setTitle("Effective Java 3rd Edition");
+        bookDao.update(book);
+        System.out.println("Updated: " + book);
 
+        bookDao.deleteById(book.getId());
```

Comment:
The `deleteById` method in `BookDaoImpl` should return a boolean value indicating the success of the operation. Ensure that it returns `true` if a book was successfully deleted and `false` otherwise, as per the checklist requirement: "Don't return `true` all the time in the method `delete`. Let's return boolean value depending on `preparedStatement.executeUpdate()` result."